src/utils.rs

```diff
@@ -20,6 +20,69 @@ pub fn abort_on_panic<T>(f: impl FnOnce() -> T) -> T {
     t
 }
 
+/// Declares unstable items.
+#[doc(hidden)]
+#[macro_export]
```

Comment:
Why are these macros exported?

I think it is enough to use `#[macro_use]` for the `utils` module if these macros are only used in this crate.
```rust
#[macro_use]
mod utils;
```

Reply:
This doesn't work - we have to keep `#[macro_export]` or else the macro won't be visible from other modules. At least that's my understanding.

Do you know how to make this work without `#[macro_export]`?

Reply:
I was able to make it work by doing the following. (see https://github.com/stjepang/async-std/pull/1)
* Apply `#[macro_use]` to the module where the macro is defined.
* Remove `crate::` from all internal macro calls.
* Remove `#[macro_export]` from the internal macro.